scripts/elder_scroll.js

```diff
@@ -72,7 +72,8 @@ ElderScroll =
 
     reachedBottom: function()
     {
-        console.log('reachedBottom');
+        // don't spam our console
+        //console.log('reachedBottom');
```

Comment:
pls stop

Reply:
Apologies. I'll undo the commented `console.log()`'s. I tend to disable unnecessary logging when I'm doing other debugging so my log is cleaner. I forgot about those particular logging bits.

Reply:
No it's cool, I mean you should delete the code entirely instead of just commenting it out.  Go for it.